src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -0,0 +1,640 @@
+            GetNode<CustomCheckBox>(BiodiversityNearbyPatchIsFreePopulationPath);
+        lowBiodiversityLimitSpinBox = GetNode<SpinBox>(LowBiodiversityLimitPath);
+        maximumSpeciesInPatchSpinBox = GetNode<SpinBox>(MaximumSpeciesInPatchPath);
+        moveAttemptsPerSpeciesSpinBox = GetNode<SpinBox>(MoveAttemptsPerSpeciesPath);
+        mutationsPerSpeciesSpinBox = GetNode<SpinBox>(MutationsPerSpeciesPath);
+        newBiodiversityIncreasingSpeciesPopulationSpinBox =
+            GetNode<SpinBox>(NewBiodiversityIncreasingSpeciesPopulationPath);
+        protectMigrationsFromSpeciesCapCheckBox = GetNode<CustomCheckBox>(ProtectMigrationsFromSpeciesCapPath);
+        protectNewCellsFromSpeciesCapCheckBox = GetNode<CustomCheckBox>(ProtectNewCellsFromSpeciesCapPath);
+        refundMigrationsInExtinctionsCheckBox = GetNode<CustomCheckBox>(RefundMigrationsInExtinctionsPath);
+        strictNicheCompetitionCheckBox = GetNode<CustomCheckBox>(StrictNicheCompetitionPath);
+        speciesSplitByMutationThresholdPopulationAmountSpinBox =
+            GetNode<SpinBox>(SpeciesSplitByMutationThresholdPopulationAmountPath);
+        speciesSplitByMutationThresholdPopulationFractionSpinBox =
+            GetNode<SpinBox>(SpeciesSplitByMutationThresholdPopulationFractionPath);
+        useBiodiversityForceSplitCheckBox = GetNode<CustomCheckBox>(UseBiodiversityForceSplitPath);
+
+        currentGenerationLabel = GetNode<Label>(CurrentGenerationLabelPath);
+        runStatusLabel = GetNode<Label>(RunStatusLabelPath);
+        finishOneGenerationButton = GetNode<Button>(RunGenerationButtonPath);
+        runOneStepButton = GetNode<Button>(RunStepButtonPath);
+        abortButton = GetNode<Button>(AbortButtonPath);
+        playWithCurrentSettingButton = GetNode<Button>(PlayWithCurrentSettingPath);
+
+        autoEvoResultsLabel = GetNode<CustomRichTextLabel>(ResultsLabelPath);
+        historyListMenu = GetNode<CustomDropDown>(HistoryListMenuPath);
+
+        speciesPreview = GetNode<SpeciesPreview>(SpeciesPreviewPath);
+        hexPreview = GetNode<CellHexPreview>(HexPreviewPath);
+        speciesListMenu = GetNode<CustomDropDown>(SpeciesListMenuPath);
+        speciesDetailsLabel = GetNode<CustomRichTextLabel>(SpeciesDetailsLabelPath);
+        evolutionaryTree = GetNode<EvolutionaryTree>(EvolutionaryTreePath);
+
+        exitConfirmationDialog = GetNode<CustomConfirmationDialog>(ExitConfirmationDialogPath);
+
+        // Init game
+        gameProperties = GameProperties.StartNewMicrobeGame(new WorldGenerationSettings());
+        autoEvoConfiguration = (AutoEvoConfiguration)SimulationParameters.Instance.AutoEvoConfiguration.Clone();
+
+        // Init generation 0 (LUCA)
+        runResultsList.Add(new LocalizedStringBuilder());
+        speciesHistoryList.Add(new Dictionary<uint, Species>
+        {
+            { gameProperties.GameWorld.PlayerSpecies.ID, gameProperties.GameWorld.PlayerSpecies },
```

Comment:
I guess this is fine for now, but could you add a TODO comment that it would be nice if we could remove the player species entirely and have an NPC species as the LUCA?

Reply:
I'm not sure whether GameWorld will go on without a player species...

Reply:
There's no option to unset player species.

Reply:
See my later comments that make this specific review comment irrelevant. If this wasn't irrelevant I would have asked you to write such methods, but turns out we don't need that.